examples/flutter_gallery/lib/demo/material/backdrop_demo.dart

```diff
@@ -301,9 +301,9 @@ class _BackdropDemoState extends State<BackdropDemo> with SingleTickerProviderSt
 
     final double flingVelocity = details.velocity.pixelsPerSecond.dy / _backdropHeight;
     if (flingVelocity < 0.0)
-      _controller.fling(velocity: math.max(2.0, -flingVelocity));
+      _controller.fling(velocity: math.max(2, -flingVelocity));
     else if (flingVelocity > 0.0)
-      _controller.fling(velocity: math.min(-2.0, -flingVelocity));
+      _controller.fling(velocity: math.min(-2, -flingVelocity));
     else
       _controller.fling(velocity: _controller.value < 0.5 ? -2.0 : 2.0);
```

Comment:
And here?

Reply:
Here I hope we can eventually push the double context into the branches. It's very annoying that we don't.